docs/index.md

```diff
@@ -184,10 +193,12 @@ Runs the comparison against the screen
 | options.blockOutStatusBar | <code>boolean</code> | blockout the statusbar yes or no, it will override the global |
 | options.blockOut | <code>object</code> | blockout with x, y, width and height values |
 | options.disableCSSAnimation | <code>boolean</code> | enable or disable CSS animation |
-| options.ignoreAntialiasing | <code>boolean</code> | compare images an discard anti aliasing |
+| options.ignoreAlpha | <code>boolean</code> | compare images and discard alpha |
+| options.ignoreAntialiasing | <code>boolean</code> | compare images and discard anti aliasing |
 | options.ignoreColors | <code>boolean</code> | Even though the images are in colour, the comparison wil compare 2 black/white images |
+| options.ignoreLess | <code>boolean</code> | compare images and compare with less (red = 16, green = 16, blue = 16, alpha = 16, minBrightness=16, maxBrightness=240) |
+| options.ignoreNothing | <code>boolean</code> | compare images and compare with less (red = 0, green = 0, blue = 0, alpha = 0, minBrightness=0, maxBrightness=255) |
 | options.ignoreTransparentPixel | <code>boolean</code> | Will ignore all pixels that have some transparency in one of the images |
```

Comment:
Why did you decide to remove saveAboveTolerance property from checkScreen method?

Reply:
Great, thanks!